<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ECMaterialsSteppableDlg</class>
 <widget class="QDialog" name="ECMaterialsSteppableDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ECMaterials Steppable: Please define extracellular material interactions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Material Interactions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Catalyst</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox_mtlC"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Reactant</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_mtlR"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Coefficient</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_mtl"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton_mtlAdd">
               <property name="text">
                <string>Add Interaction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton__mtlDel">
               <property name="text">
                <string>Delete Interaction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton__mtlClear">
               <property name="text">
                <string>Clear Table</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_mtl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <column>
            <property name="text">
             <string>Catalyst</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Reactant</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Coefficient</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Field Interactions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Catalyst</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Reactant</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Coefficient</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox_fldC"/>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_fldR"/>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_fld"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButton_fldAdd">
               <property name="text">
                <string>Add Interaction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_fldDel">
               <property name="text">
                <string>Delete Interaction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_fldClear">
               <property name="text">
                <string>Clear Table</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_fld">
           <column>
            <property name="text">
             <string>Catalyst</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Reactant</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Coefficient</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Cell Interactions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>ECMaterial</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Cell Type</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Response Type</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Coefficient</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox_cellM"/>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_cellT"/>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="comboBox_cellR"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_cell"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>New Cell Type: </string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="comboBox_cellT_New"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="pushButton_cellAdd">
               <property name="text">
                <string>Add Interaction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_cellDel">
               <property name="text">
                <string>Delete Interaction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_cellClear">
               <property name="text">
                <string>Clear Table</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_cell">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <column>
            <property name="text">
             <string>ECMaterial</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Cell Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Response Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Coefficient</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Cell Type</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Material Diffusion</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget_diff">
         <column>
          <property name="text">
           <string>ECMaterial</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Diffuses</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Coefficient</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
